app/assets/stylesheets/boards.scss

```diff
@@ -0,0 +1,27 @@
+ 	.board {
+ 		width: 60%;
+    	height: 70%;
+    	border: 3px solid #333;
+    	margin: 10px auto 10px auto;
+
```

Comment:
I just tried visiting this view in dev, and the alignment between the `.board` div and the table inside it is very brittle because the former uses `%` to adjust to the size of the viewport while the latter consists of fixed-size cells. At larger viewport sizes the border on the div goes out way past the board, while at small sizes, the div constrains the board and distorts it into a rectangle. 

This can be addressed by turning the wrapping div into a basic Bootstrap container and applying all the relevant properties (border and margin) directly to the table.